src/clue.js

```diff
@@ -1,94 +1,168 @@
+  first_name: "Kasandra",
+  last_name: "Scarlet",
+  color: "red",
+  description: "She is an A-list movie star with a dark past",
+  age: 31,
+  image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+  occupation: "Actor"
+};
 
-mrMustard
-first_name:   Jack
-last_name:    Mustard
-color:        yellow
-description:  He is a former football player who tries to get by on his former glory
-age:          62
-image:        https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-occupation:   Retired Football player
+var mrsPeacock = {
+  first_name: "Eleanor",
+  last_name: "Peacock",
+  color: "blue",
+  description:
+    "She is from a wealthy family and uses her status and money to earn popularity",
+  age: 36,
+  image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+  occupation: "Socialité"
+};
 
-// Weapons
+var mrMustard = {
+  first_name: "Jack",
+  last_name: "Mustard",
+  color: "yellow",
+  description:
+    "He is a former football player who tries to get by on his former glory",
+  age: 62,
+  image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation: "Retired Football player"
+};
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+// Weapons Collection
+var rope = {
+  name: "rope",
+  weight: 10
+};
+var knife = {
+  name: "knife",
+  weight: 8
+};
+var candlestick = {
+  name: "candlestick",
+  weight: 2
+};
+var dumbbell = {
+  name: "dumbbell",
+  weight: 30
+};
+var poison = {
+  name: "poison",
+  weight: 2
+};
+var axe = {
+  name: "axe",
+  weight: 15
+};
+var bat = {
+  name: "bat",
+  weight: 13
+};
+var trophy = {
+  name: "trophy",
+  weight: 25
+};
+var pistol = {
+  name: "pistol",
+  weight: 20
+};
 
-// Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+var weaponsArray = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy
+];
 
-// Characters Collection
-var charactersArray = [];
+var charactersArray = [
+  mrGreen,
+  drOrchid,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  mrMustard
+];
+var roomsArray = [
+  "Dinning Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio"
+];
 
-// Rooms' Collection
-var roomsArray = [];
+// Find a random element of the array - randomSelector
 
-// Weapons Collection
-var weaponsArray = [];
+//Defines randomSelector
+
+function randomSelector() {
```

Comment:
There is a more efficient way to create this function passing the array as a parameter right? I think now you had the chance to see it during the review.